api/envoy/config/core/v3/protocol.proto

```diff
@@ -224,6 +224,11 @@ message HttpProtocolOptions {
   // <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.drain_timeout>`.
   google.protobuf.Duration max_connection_duration = 3;
 
+  // The maximum response headers size allowed from the upstream server. If not set, the default max response headers
+  // size allowed is 80 KiB. Response that exceed this limit would fail with a protocol error.
```

Comment:
As you note the default value is different for H1 (60K) and H2/H3 (80K).
IIUC this is intentional (lower limits of the total size of the headers when multiple streams are used).
@yanavlasov do you think the value should be the same for all HTTP versions or should it be different (as in the current implementation)?

Reply:
Do we need to make it lower by default or is it okay to keep it consistent and set it to **80K** by default?

Reply:
It depends on the reasoning behind the 2 values.
@yanavlasov WDYT?